src/adapters/Base/index.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosRequestConfig } from 'axios';
+
+import { JSONObject, keysToSnakeCase } from 'helpers/json';
+import requestManager from 'lib/request/v1/requestManager';
+
+export function get(path: string, params?: JSONObject) {
+  const requestOptions: AxiosRequestConfig = {};
+  if (params) {
+    requestOptions.params = keysToSnakeCase(params);
```

Comment:
With `snakecase-keys`, you wouldn't need to implement any helper:

```suggestion
    requestOptions.params = snakecaseKeys(params);
```